package.json

```diff
@@ -0,0 +1,73 @@
+{
+  "name": "ffc-ce-admin-web",
+  "description": "Digital service for calculation engine.",
+  "version": "0.0.1",
+  "homepage": "https://github.com/DEFRA/ffc-ce-admin-web",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/DEFRA/ffc-ce-admin-web.git"
+  },
+  "main": "index.js",
+  "scripts": {
+    "build": "node-sass --output-style=expanded --output=server/public/build/stylesheets client/sass",
+    "build:watch": "node-sass --output-style=expanded --output=server/public/build/stylesheets --watch client/sass",
+    "pretest": "npm run lint",
+    "lint": "standard",
+    "test": "jest",
+    "test:watch": "nodemon --inspect=0.0.0.0 --ext js,njk --legacy-watch --exec npm test",
+    "start:watch": "npm-run-all --parallel build:watch start:nodemon",
+    "start:nodemon": "nodemon --inspect=0.0.0.0 --ext css,js,njk --legacy-watch index.js"
+  },
+  "author": "Defra DDTS",
+  "contributors": [
+    "John Watson john.watson2@rpa.gov.uk",
+    "Mark Harrop mark.harrop@defra.gov.uk",
+    "Matthew Collins matthew.collins@defra.gov.uk",
+    "David Bingham david.bingham@defra.gov.uk",
+    "Neil McLaughlin neil.mclaughlin@defra.gov.uk",
+    "Paul Andrews paul.andrews@defra.gov.uk",
+    "Phil Benson phil.benson@defra.gov.uk"
```

Comment:
The email addresses need angle brackets around them. <>
`Matthew Collins <matthew.collins@defra.gov.uk>`
Trivial thing to fix, but while new people are being added it's best to get it right.